src/pulse3D/excel_writer.py

```diff
@@ -194,6 +196,7 @@ def write_xlsx(
     prominence_factors: Tuple[Union[int, float], Union[int, float]] = DEFAULT_PROMINENCE_FACTORS,
     width_factors: Tuple[Union[int, float], Union[int, float]] = DEFAULT_WIDTH_FACTORS,
     peaks_valleys: Dict[str, List[List[int]]] = None,
+    include_stim_protocols: bool = False,
```

Comment:
Last thing we need to do is add some unit tests for this new param. We at least want to test one file with stim protocols and one without